src/main/kotlin/org/jetbrains/research/ml/codetracker/data/UiData.kt

```diff
@@ -37,16 +38,17 @@ class UiData(val tasks: List<Task>) {
             uiValue = defaultUiValue
         }
 
-        fun isDefault() : Boolean {
+        fun isDefault(): Boolean {
             return uiValue == defaultUiValue
         }
 
-        fun isDefault(new: T) : Boolean {
+        fun isDefault(new: T): Boolean {
             return new == defaultUiValue
         }
     }
 
-    fun tasksNames() : List<String> = tasks.map { it.name }
+    // Todo: get real list by language
+    fun tasksNames(): List<String> = listOf("pies")
```

Comment:
Let's not include dummy lists in PR, especially when we have the real one. I thought, that the previous version with the map is okay, the language switch is my part of job and I will implement it :)